conf/app.routes

```diff
@@ -40,3 +40,6 @@ GET        /check-another-guarantee-balance         controllers.BalanceConfirmat
 GET        /manage-transit-movements                controllers.BalanceConfirmationController.manageTransitMovements()
 
 GET        /details-do-not-match                    controllers.DetailsDontMatchController.onPageLoad()
+
+GET        /unsupported-guatantee-type              controllers.UnsupportedGuaranteeTypeController.onPageLoad()
+POST       /unsupported-guatantee-type              controllers.UnsupportedGuaranteeTypeController.onSubmit()
```

Comment:
Small misspelling for guarantee on the routes

Reply:
Oops. Fixed